### PR TITLE
GT-134 Add annotation to change architecture

### PR DESCRIPTION
**Pull request by jwierzbo**

### CODE REVIEW

**Review by nikita-vanyasin — APPROVED** [review]
LGTM. But are there any integration tests for this feature?

**Comment by jwierzbo** [comment]
@nikita-vanyasin https://github.com/arangodb/kube-arangodb-test/pull/130/files

File: pkg/deployment/reconcile/action_set_current_arch.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
+//
```
Comment (ajanikow) [inline pkg/deployment/reconcile/action_set_current_arch.go:1]: File name should match action name (validator will ensure it soon)

File: pkg/deployment/rotation/check.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ type Mode int
 
 const (
 	SkippedRotation Mode = iota
-	SilentRotation
+	SilentRotation       // Propagate changes without restart
```
Comment (ajanikow) [inline pkg/deployment/rotation/check.go:39]: Add comment in go format

**Review by ajanikow — CHANGES_REQUESTED** [review]

**Review by adam-janikowski — APPROVED** [review]

**Pull request merged**